packages/transaction-controller/src/TransactionController.ts

```diff
@@ -944,6 +945,33 @@ export class TransactionController extends BaseController<
     this.update({ transactions: this.trimTransactionsForState(transactions) });
   }
 
+  /**
+   * Add params to transactionMeta.
+   *
+   * @param transactionMetaId - ID of the transaction.
+   * @param securityAlertResponse - New params to be added to tarnsaction meta.
+   */
+  updateSecurityAlertResponse(
+    transactionMetaId: string,
+    securityAlertResponse: SecurityAlertResponse,
+  ) {
+    if (!transactionMetaId || !securityAlertResponse) {
```

Comment:
Rather than quietly allowing them bad inputs, should we let the subsequent find error detect any bad IDs?

Equally do we want to validate the alert response exists and throw otherwise?

Or should we make it optional in order to let this method clear it if needed?